src/PoshArmDeployment/Public/ResourceProviders/Microsoft.Insights/Add-ArmApplicationInsightsMetricAlertCriteriaODataType.ps1

```diff
@@ -0,0 +1,18 @@
+function Add-ArmApplicationInsightsMetricAlertCriteriaODataType {
+    [CmdletBinding(SupportsShouldProcess = $true)]
+    [OutputType("ApplicationInsightsMetricAlert")]
+    Param(
+        [PSTypeName("ApplicationInsightsMetricAlert")]
+        [Parameter(Mandatory, ValueFromPipeline)]
+        $MetricAlert,
+        [string]
+        [ValidatePattern("^(\[.*\]|[a-zA-Z0-9-.]*)$")]
+        $ODataType = "Microsoft.Azure.Monitor.SingleResourceMultipleMetricCriteria"
```

Comment:
We currently only have 2  possible options `"Microsoft.Azure.Monitor.SingleResourceMultipleMetricCriteria"` and `"Microsoft.Azure.Monitor.MultipleResourceMultipleMetricCriteria"`. Should we use a `[switch]` instead?